src/content/en/fundamentals/getting-started/service-worker/register-and-install.markdown

```diff
@@ -0,0 +1,58 @@
+---
+layout: shared/narrow
+title: "How to Register and Install a Service Worker"
```

Comment:
Title is a bit confusing, this just covers the register, install is covered in the next doc.